components/table.js

```diff
@@ -0,0 +1,16 @@
+import { Table } from "antd";
+
+export default function AppTable({ isLoading, columns, dataSource, expandable, className, pagination }) {
+  return (
+    <Table
+      scroll={{ x: 1000 }}
+      loading={isLoading}
+      rowKey="id"
```

Comment:
```suggestion
      rowKey={ rowKey || "id" }
```

please take rowKey as props as well

Reply:
implemented, please have a look.